crates/bootstrap_srv/src/test.rs

```diff
@@ -151,7 +161,19 @@ impl<'lt> PutInfo<'lt> {
             },
         );
 
-        match ureq::put(&addr).send_string(&info) {
+        let agent_builder = ureq::builder();
+        let agent = if self.use_tls {
+            let client_config = ClientConfig::builder()
+                .dangerous()
+                .with_custom_certificate_verifier(Arc::new(NoVerify))
```

Comment:
Just want to test with self-signed certificates. Server validity not relevant in a test. It'd be nice if they'd expose this for testing but they only seem to do this inside their own tests and expect everyone else to use valid certs. Could also try to provision certs as part of the test but doesn't seem that helpful

Reply:
Yep, I've had to do this exact thing multiple times too... indeed wish there was a better way.